actions/oh_shit_error.yaml

```diff
@@ -0,0 +1,38 @@
+---
+name: oh_shit_error
+description: Send out an oh shit error
+enabled: true
+runner_type: mistral-v2
+entry_point: workflows/oh_shit_error.yaml
+pack: arteria-packs
+parameters:
+
+  context:
+    default: {}
+    immutable: true
+    type: object
+
+  workflow:
+    default: arteria-packs.oh_shit_error
+    immutable: true
+    type: string
+
+  host:
+    default: null
+    required: true
+    type: string
+    description: "Host where the runfolder is located."
+
+  runfolder:
+    default: null
+    type: string
+    required: true
+    description: "Name of runfolder that failed"
+
+  send_mail_to:
+    default: johan.hermansson@medsci.uu.se
```

Comment:
Should we change the default address here (and in other places) to examples so you and our list don't have to be spammed if some other user picks this up?